### ISSUE TITLE
Ability to group against different seed values

### ISSUE THREAD

**Issue by SGevorg** [aimhubio/aim#334]
When running multiple experiments with the same set of hyper-parameters but different seeds, there is a need to be able to verify whether the given set of hyperparameters are doing well regardless of seed value or not.

Grouping against the seed or grouping by every param but the seed would allow to clearly see the trends of metrics across different seeds for given set of hyperparameters.
